### PR TITLE
[MIG] mrp_sale_info: Migration to V15.0

### PR DESCRIPTION
**Pull request by BlommaertsEdwin**
Standard Migration, 
- changed version numbers
- changed test to use Transaction case

### CODE REVIEW

**Comment by BlommaertsEdwin** [comment]
Test failed on module : "TestBoM.test_replenishment"
Which is unrelated to my changes.

**Review by ajaniszewska-dev — APPROVED** [review]

File: mrp_sale_info/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {
     "name": "MRP Sale Info",
     "summary": "Adds sale information to Manufacturing models",
-    "version": "14.0.1.1.0",
+    "version": "15.0.0.1.0",
```
Comment (HviorForgeFlow) [inline mrp_sale_info/__manifest__.py:8]: First migrated version is 15.0.1.0.0

File: mrp_sale_info/__manifest__.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
     "category": "Manufacturing",
     "website": "https://github.com/OCA/manufacture",
     "author": "AvanzOSC, " "Tecnativa, " "Odoo Community Association (OCA)",
```
Comment (HviorForgeFlow) [inline mrp_sale_info/__manifest__.py:11]: Drop useless quotes on same line

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
Could you verify using test that work orders are created with proper sale information? Thanks for your contribution

**Pull request closed without merging**